Tools/Make/Make.PeleLM

```diff
@@ -117,6 +117,9 @@ SKIP_NS_SPECIFIC_CODE = TRUE
 # IAMR
 Bdirs += $(IAMR_HOME)/Source
 Bdirs += $(IAMR_HOME)/Source/Src_$(DIM)d
+ifeq ($(USE_EB), TRUE)
+   Bdirs   += $(IAMR_HOME)/Source/MOL
+endif
```

Comment:
Do we not want MOL for non-EB?

Reply:
we don't want MOL, and in the long run 2nd order Godunov will replace MOL for EB I think

Reply:
Ultimately we want Godunov, but is MOL just not coded for non-EB?